src/pages/ReceiverDetailsEdit.tsx

```diff
@@ -363,23 +382,29 @@ export const ReceiverDetailsEdit = () => {
                       value={receiverEditFields.yearMonth}
                       onChange={handleDetailsChange}
                     />
-                    <Input
-                      id="dateOfBirth"
-                      name="dateOfBirth"
-                      label={
-                        <InfoTooltip
-                          hideTooltip={
-                            !isVerificationFieldConfirmed("DATE_OF_BIRTH")
-                          }
-                          infoText={alreadyConfirmedText}
-                        >
-                          {VerificationFieldMap["DATE_OF_BIRTH"]}
-                        </InfoTooltip>
-                      }
-                      fieldSize="sm"
-                      value={receiverEditFields.dateOfBirth}
-                      onChange={handleDetailsChange}
-                    />
+                    <div className="Input Input--sm">
+                      <CheckMark
+                        hideCheckMark={
+                          !isVerificationFieldConfirmed("DATE_OF_BIRTH")
+                        }
+                        infoText={alreadyConfirmedText}
+                      >
+                        {VerificationFieldMap["DATE_OF_BIRTH"]}
+                      </CheckMark>
+                      <DatePicker
+                        id="dateOfBirth"
+                        name="dateOfBirth"
+                        selected={selectedDob}
+                        onChange={handleDateChange}
+                        dateFormat="yyyy-MM-dd"
+                        placeholderText="Select a date"
+                        showMonthDropdown
+                        showYearDropdown
+                        scrollableYearDropdown
+                        yearDropdownItemNumber={100}
+                        maxDate={new Date()}
+                      />
```

Comment:
Q: what's the adbantage of adding this new dependency instead of just setting the input type to `date`?